src/exercises/Exercises.js

```diff
@@ -194,9 +238,18 @@ export default function Exercises({
     setExerciseSession(exerciseSession);
   }
 
+  // Standard flow when user completes exercise session
   if (finished) {
     api.logReaderActivity(api.COMPLETED_EXERCISES, articleID, "", source);
```

Comment:
use `logUserActivity` instead of readerActivity. Reader is for the article "reader" part of the system.